SYCL/ESIMD/unused_spec_const.cpp

```diff
@@ -1,5 +1,5 @@
 // REQUIRES: gpu
-// UNSUPPORTED: cuda || hip
+// UNSUPPORTED: cuda || hip || esimd_emulator
```

Comment:
Please add `esimd_emulator` as `XFAIL:`, like https://github.com/intel/llvm-test-suite/blob/5eeb87edd4df7950dd2065d26a20029f1888e4f1/SYCL/ESIMD/spec_const/spec_const_bool.cpp#L13

And, there is a folder for 'spec_const' tests - `$ROOT/SYCL/ESIMD/spec_const`. Please move this file to the folder.

Reply:
Ok, fixed. Thank you. Please see: 1f933c63916b434f57b7e066d9feb3ab3437f7e8